### ISSUE TITLE
Release target missing the git tag  info

### ISSUE THREAD

**Issue by BohuTANG** [datafuselabs/databend#5413]
**Summary**
Now, the databend release version doesn't contain `tag`, for example, `0.7.49-nightly`:
```
mysql> select version();
+--------------------------------------------------------------------------+
| @@version()                                                              |
+--------------------------------------------------------------------------+
| 8.0.26--b97c58f-simd(rust-1.61.0-nightly-2022-05-16T16:24:57.033776992Z) |
+--------------------------------------------------------------------------+
```

`8.0.26-[tag info not takes here]-b97c58f-simd(rust-1.61.0-nightly-2022-05-16T16:24:57.033776992Z) `

The server log:
```
2022-05-17T00:43:10.306421Z  INFO databend_query: DatabendQuery -b97c58f-simd(rust-1.61.0-nightly-2022-05-16T16:24:57.033776992Z)
```
https://github.com/datafuselabs/databend/blob/b97c58fd3258b43c1b2b8dbb4d83b4f2db1d2033/query/src/version.rs#L22-L31

**Comment by DCjanus** [comment]
Looks like this bug has been fixed in https://github.com/datafuselabs/databend/commit/fb81d8277563e29919d3a6fd2e62300ed6ead90c

**Comment by BohuTANG** [comment]
> Looks like this bug has been fixed in [fb81d82](https://github.com/datafuselabs/databend/commit/fb81d8277563e29919d3a6fd2e62300ed6ead90c)

It fixes the build from the source on your local, but the github release build does not work. For example:
https://github.com/datafuselabs/databend/releases/tag/v0.7.55-nightly
```
./databend-query
... ...
2022-05-21T00:54:07.520672Z  INFO databend_query: DatabendQuery -5ed5323-simd
```

Maybe it is related to the release script(@everpcpc  for verfiy):
https://github.com/datafuselabs/databend/blob/main/.github/actions/build_release/action.yml

**Comment by DCjanus** [comment]
My mistake, this bug has been fixed in [MacOS build](https://github.com/datafuselabs/databend/releases/download/v0.7.55-nightly/databend-v0.7.55-nightly-aarch64-apple-darwin.tar.gz), but I've not check in other targets.

```
2022-05-21T01:59:57.381302Z  INFO databend_query: DatabendQuery v0.7.55-nightly-5ed5323-simd(rust-1.63.0-nightly-2022-05-20T16:24:54.246883Z)
```
